rm-datasource/src/main/java/io/seata/rm/datasource/undo/mysql/MySQLUndoUpdateExecutor.java

```diff
@@ -58,10 +58,9 @@ protected String buildUndoSQL() {
         Field pkField = row.primaryKeys().get(0);
         List<Field> nonPkFields = row.nonPrimaryKeys();
         String updateColumns = nonPkFields.stream()
-            .map(field -> keywordChecker.checkAndReplace(field.getName()) + " = ?")
+            .map(field -> field.getName() + " = ?")
```

Comment:
mark: update columns must not contain escape, otherwise the original SQL will report an error.